### DISCUSSION TITLE
Video transcoding result in 120fps frame rate

### DISCUSSION THREAD

**Discussion by mingqlin** [immich-app/immich#7284] [Feature Request]
### The bug

I have some video are shot in 1080P 120fps, it turn out, the video transcoding export video in 720P 120fps, I would recommend transcoded video to be in 720P 30fps to get smaller video size. 

Also allow video player to have option to choose original quality or transcoded quality

### The OS that Immich Server is running on

Unraid

### Version of Immich Server

v1.94.1

### Version of Immich Mobile App

v1.94.1

### Platform with the issue

- [X] Server
- [X] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
Unraid
```

### Your .env content

```Shell
Unraid
```

### Reproduction steps

```bash
See above
```

**Comment by mertalev** [comment]
The tricky thing here is that slow-motion videos have high fps and an open feature request is to play them back in slow-motion as well. So if we transcode to a lower frame rate, that feature won't be possible anymore. There's probably some metadata we can use to determine if a video is meant to be slow-motion and only cap the fps if it isn't. But I'm not sure it's worth the added complexity.

**Reply by kevincox** [reply]
In this proposal https://github.com/immich-app/immich/discussions/7284#discussioncomment-16447728 users who wanted this could configure their minimum framerate lower. So if you wanted smooth slow motion you could give yourself some buffer.

**Comment by kevincox** [comment]
For for my use case I often have 119.88 or 239.76fps video and would like to reduce that to ~30fps. For me a policy like "the lowest even divisor that is >24fps" would be great. So both of these would be cut down to 29.97fps video. A slightly simpler policy would be to keep halving the framerate until the next having would bring it under 24fps. 

IMHO this should probably be the default behaviour, but it would be nice to have it configurable. Probably `acceptedMaxFramerate` and  `targetMinFramerate` values would be suitable. In my situation I would probably set them to 60 and 24 respectively. If checking the defaults is undesirable they can both be set to infinity (likely represented as `null` or `0`) to not have any effect.

I'm also assuming that very few people want interpolation or uneven frame timings. But it would be possible to add in the future. Just add an option for how to achieve the framerate and instead of diving frames `targetMinFramerate` is hit exactly via interpolation or pulldown-like techniques. So I don't think that needs to be considered in the first version.
